Release/include/cpprest/asyncrt_utils.h

```diff
@@ -649,6 +658,8 @@ class datetime
 
     bool is_initialized() const { return m_interval != 0; }
 
+    static datetime maximum() { return datetime(static_cast<interval_type>(-1)); }
```

Comment:
>does this work if the interval type is signed? 

We know it isn't signed. `typedef uint64_t interval_type;`

>Why do this instead of using numeric_limits?

Because that would require dragging in `<limits>`.

Reply:
I don't love depending on the typedef being unsigned, but it's pre-existing (other stuff in datetime does the same thing), so I don't have a huge problem keeping it as-is.